### DISCUSSION TITLE
do i use my wyze account creds or rtsp creds?

### DISCUSSION THREAD

**Discussion by ljmerza** [mrlt8/docker-wyze-bridge#54] [Q&A]
I've tried using my wyze creds and my local camera creds but both return `400 Client Error: Bad Request for url: https://auth-prod.api.wyze.com/user/login`

**Comment by mrlt8** [comment]
IIRC, 400 usually means Invalid credentials.

You need to use the wyze account associated with your cameras.
Double check your docker-compose.yml for any typos.
